pages/linux/chroot.md

```diff
@@ -0,0 +1,23 @@
+# chroot
+
+> Run command or interactive shell with special root directory.
+
+- General Syntax
+
+`chroot [OPTION] NEWROOT {{COMMAND [ARG...]}}`
+
+- specify user and group (ID or name) to use
+
+`chroot −−userspec={{USER:GROUP}}`
+
+- specify supplementary groups as g1,g2,..,gN
```

Comment:
I had a peak at [usermod](https://github.com/tldr-pages/tldr/blob/2c215438cd3f57379fb57a8d8173897941ee8caa/pages/linux/usermod.md) where we just use `{{group1,group2}}`. I like that best, so you can safely update to that. Also keep in mind everything is lowercase, except where it's required to be caps.